### PR TITLE
All conventions are followed in resources files

### LINKED ISSUE #898: Make mentioned changes in resources files to follow the same convention
## Description
As a contributor,
I need consistent style followed in resources file,
so that I don't get confused that whether some part of the code is missing or not.

## Mocks
https://github.com/anitab-org/mentorship-backend/blob/6fe6929fed2007ed43242f36c44f9bcca1d79979/app/api/resources/user.py#L171-L182
Here the route is below the responses which breaks the convention.
https://github.com/anitab-org/mentorship-backend/blob/6fe6929fed2007ed43242f36c44f9bcca1d79979/app/api/resources/user.py#L247-L268
Here the responses aren't attached with the route function but with the put().

### Update [Required]
- [ ] Make the above mentioned changes in the code to follow the same convention throughout the file.
- [ ] Check the same with other resources files and fix them if such convetion is broken.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
4 hours

### PR DESCRIPTION
**Pull request by gitpurva**
### Description

Mentioned changes in the resources file to follow same convention.

Fixes #898

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### CODE REVIEW

**Comment by devkapilbansal** [comment]
Continue of #905 
@PrashanthPuneriya please review it too

File: app/api/resources/admin.py [hunk 1]
```diff
@@ -77,20 +78,21 @@ def post(cls):
...
             f"{messages.AUTHORISATION_TOKEN_IS_MISSING['message']}"
         }
     )
-    @admin_ns.response(HTTPStatus.FORBIDDEN, f"{messages.USER_IS_NOT_AN_ADMIN}")
-    @admin_ns.expect(auth_header_parser)
+@admin_ns.expect(auth_header_parser)
+class ListAdmins(Resource):
+    @classmethod
+    @jwt_required
+
```
Comment (isabelcosta) [inline app/api/resources/admin.py:95]: this empty line isn't needed right?

**Comment by devkapilbansal** [comment]
@gitpurva there are some merge conflicts, please resolve them

**Comment by NenadPantelic** [comment]
HI, @gitpurva do you want to finish this PR or it should be closed?

**Comment by devkapilbansal** [comment]
Closing due to inactivity. @gitpurva feel free to ping again if you want to continue

**Pull request closed without merging**